python/hawkey/goal-py.cpp

```diff
@@ -110,17 +110,18 @@ static int
 args_run_parse(PyObject *args, PyObject *kwds, int *flags, PyObject **callback_p)
 {
     const char *kwlist[] = {"callback", "allow_uninstall", "force_best", "verify",
-        "ignore_weak_deps", "ignore_weak", NULL};
+        "ignore_weak_deps", "ignore_weak", "dont_protect_running_kernel", NULL};
     int ignore_weak = 0;
     int ignore_weak_deps = 0;
     int allow_uninstall = 0;
     int force_best = 0;
     int verify = 0;
+    int dont_protect_running_kernel = 0;
     PyObject *callback = NULL;
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|Oiiiii", (char**) kwlist,
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|Oiiiiii", (char**) kwlist,
```

Comment:
What about to use new keyword flags that could be shared for multiple options? See https://github.com/rpm-software-management/libdnf/pull/880/commits/2ade7f18adf0eb1d119dde6e9c1cb2c360a30c48

Reply:
Absolutely. I wanted to build on top of your PR when it gets merged. Btw, version bump is also missing.